### DISCUSSION TITLE
Feature request: add config option <ConnectKitButton showDontHaveAWallet={false} />

### DISCUSSION THREAD

**Discussion by ryanberckmans** [family/connectkit#114] [Q&A] [answered]
I'm requesting a config option to hide the modal section, "I don't have a wallet"

<img width="232" alt="image" src="https://user-images.githubusercontent.com/299324/217870269-c70bd319-046c-40af-b328-9774be5e08ae.png">

**Comment by lochie** [comment, marked as answer]
@ryanberckmans you can find this option under [`hideNoWalletCTA`](https://docs.family.co/connectkit/api-reference#ConnectKitProvider-hideNoWalletCTA) 👍

**Reply by ryanberckmans** [reply]
Great, thanks!

fwiw, I've spent a lot of time exploring connectkit's documentation on desktop, but entirely missed this section of variables because I didn't dig into the Theming section as I thought it was colors/visual style only and not also UX options.

**Reply by lochie** [reply]
Ah okay, I'll relay that to the team and we'll see if we can make this more visible.
Currently we've just got it organised under the [ConnectKitProvider](https://docs.family.co/connectkit/api-reference#connectkitprovider) reference since it goes into it's `options` property, but the above `customThemes` section does look a little close.

Thanks for the feedback!
